docs/en/connector-v2/source/MongoDB.md

```diff
@@ -66,6 +71,7 @@ mongodb {
     uri = "mongodb://username:password@127.0.0.1:27017/mypost?retryWrites=true&writeConcern=majority"
     database = "mydatabase"
     collection = "mycollection"
+    matchQuery = "{"id":3}"
```

Comment:
I'm not sure the json in hocon value can be parse normally, maybe you should change a way, like use object
```
matchQuery = {
     "id"=3
}
```

Reply:
This place is a string, except that this string has to conform to a certain format in order to be transformed into a BsonDocument

Reply:
The core is this method：
.find(Optional.ofNullable(params.getMatchQuery()).isPresent() ? BsonDocument.parse(params.getMatchQuery()) : new BsonDocument())
I have tested  in my development env example mode, it is ok

Reply:
> The core is this method： .find(Optional.ofNullable(params.getMatchQuery()).isPresent() ? BsonDocument.parse(params.getMatchQuery()) : new BsonDocument()) I have tested in my development env example mode, it is ok

Because I consider one thing is: the query statement is best to conform to the native syntax of mongodb, this will be very friendly to the developer, he does not need to learn anything extra, directly paste the history of mongodb query statement can be used

Reply:
Ok for me, please update e2e test case. Thanks.

Reply:
Thanks for your review,I have made the changes.